### PR TITLE
CF-10 Home Page

### PR DESCRIPTION
**Pull request by mrabiabrn**
Proposed changes
----------------

* Infinite Scroll
* Empty Home page message
* End of Home page message

Types of changes
----------------

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: Columbus/frontend/src/pages/Home/Home.js [hunk 1]
```diff
@@ -42,12 +45,41 @@ const useStyles = makeStyles((theme) => ({
                   tags: ['cat','love','terminal'],
                   multimedia: ['https://images.pexels.com/photos/8264394/pexels-photo-8264394.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940']   
                   }
+    const post3 = {owner_username : 'tarkankuzu',
```
Comment (onurcanavci) [inline Columbus/frontend/src/pages/Home/Home.js:48]: Bu constantları sayfa içerisinde tutmak yerine, Home folderı içerisine Home.constants.js dosyası oluşturulabilir. Eğer öyle olursa bu sayfadaki code u okumak daha kolay olur, nasıl taşınacağına dair bir soru olursa yardımcı olabilirim

Comment (mrabiabrn) [reply Columbus/frontend/src/pages/Home/Home.js:48]: I will put dummy data in a separate file. Thanks for the suggestion!

**Review by erencanuysal — APPROVED** [review]
I think this is good  development for now. Later, we can change colors and set a theme.

**Review by onurcanavci — APPROVED** [review]
It looks better. Thanks for changes 🚀

**Review by CodeBlo — APPROVED** [review]

**Pull request merged**